toolkit/javascripts/analytics/_init.js

```diff
@@ -0,0 +1,10 @@
+(function(root) {
+  "use strict";
+  // We've copied this from the Briefs FE
```

Comment:
This comment won't make sense as soon as the briefs PR lands, so it may as well not be there :)

Reply:
Yep, that was just to stop us getting mixed up with the duplicate code in the FE apps - that will get taken out before merging 😸